gradio/chat_interface.py

```diff
@@ -130,9 +142,10 @@ def __init__(
                             self.textbox = Textbox(
                                 container=False,
                                 show_label=False,
+                                label="Message",
```

Comment:
Why are we adding a label here if `show_label` is false?

Reply:
Because the label is used as the header of the table when examples are provided with additional inputs